main.js

```diff
@@ -58,6 +68,23 @@ class Game {
     }
 }
 
+class Timer {
+    constructor() {
+    }
+    startTimer() {
+        let pageTimer=document.getElementById("timer");
+        let numSec = 0;
+        return setInterval(function(){
+            numSec++;
+            let minutes=Math.floor(numSec/60);
+            if(minutes<10) minutes='0'+ minutes;
+            let seconds=numSec % 60;
+            if(seconds<10) seconds='0'+ seconds;
+            let timerValue=`${minutes}:${seconds}`;
+            pageTimer.firstChild.data=timerValue;
+        },1000);
+    }

```

Comment:
Let's add
- `stopTimer` method which will stop the setInterval by `intervalId`
- `isStarted` [**getter** ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)that will return status of the timer
And use them in `Game` class